views/tx.tmpl

```diff
@@ -266,19 +266,15 @@
                   </td>
                   <td class="text-right medium-sans text-nowrap pr-2 py-2"><span class="d-none d-sm-inline">Spending</span><span class="d-sm-none">Sp.</span> Tx:</td>
                   <td class="text-left py-1 text-secondary" data-target="tx.spendingTx">
-                    {{$spendingTx := index .SpendingTxns 0}}
+                    {{- $spendingTx := index .SpendingTxns 0 -}}
                     {{if $spendingTx.Hash}}
-                      <a href="/tx/{{$spendingTx.Hash}}"
-                      >{{if eq .TicketInfo.PoolStatus "missed"}}revocation{{else}}vote{{end}}
-                      </a>
-                    {{else if ne .TicketInfo.SpendStatus ""}}
-                      {{.TicketInfo.SpendStatus}}
-                    {{else}}
-                      unspent
-                    {{end}}
+                    <a href="/tx/{{$spendingTx.Hash}}">
+                      {{- if or (eq .TicketInfo.PoolStatus "missed") (eq .TicketInfo.PoolStatus "expired") -}}revocation{{- else -}}vote{{- end -}}
```

Comment:
This is fine, but note that the white space eating dashes in ` -}}revocation{{- else -}}vote{{- ` don't do anything because you've removed the whitespace from the template.  The nice thing about the dashes is that it lets you keep templates readable while removing the whitespace in the rendered html.